jme3-terrain/src/main/resources/Common/MatDefs/Terrain/PBRTerrain.frag

```diff
@@ -417,6 +425,16 @@ gl_FragColor.rgb = vec3(0.0);
     
      
     float ndotv = max( dot( normal, viewDir ),0.0);
+    #ifdef SPECULAR_AA
+        float sigma = 0.5f;
```

Comment:
The default values are set twice, here and in the j3md.
Also, can you remove the "f" suffix? Since it is unsupported in older specs, thanks

Reply:
I will modify the code later, thanks for pointing it out.